### DISCUSSION TITLE
Renaming Cities should be allowed, but also an accomplishment.

### DISCUSSION THREAD

**Discussion by JezzyBelle** [voxeltycoon/issues#382] [General]
I really want to rename some of my cities. But it would be just too easy and unrealistic to rename at will. We ARE dealing with citizens. I am not the owner of a city. I can be its benefactor though. They may even offer of their own accord to rename it after me if I have made the super happy.

I should be able to bribe or influence local politicians via providing good business (thus their happiness with me) or cash payments to lobby for changing a city name. Should not be easy, but doable. Something to represent time and hassle involved with the renaming. You do not just rename it and reprint signs in one day. It takes time to rename a city even if everyone wants to change it. 

I can see the population fight me on it to the point that I obsess and sink tons of hard cash into the effort. I probably would have had better luck had I serviced their businesses better. Finally, two years later and 2 million dollars poorly spent lobbying for it will finally see the city of Anchorage willing to discuss it. They finally offer me some choices to name it after:  my character for an addition 500,000, some new random name they choose for 100,000, or, if I insist, they will rename it to what I want... But they want more large cash and perhaps a business contract.
So now... Renaming a city is actually an accomplishment requiring cash, time, and cultivating happiness in the populace. I could also see it reverting to the original name IF , as part of the condition was that you would maintain a certain business of thiers with adequate supplies for the next year.. or so.

**Comment by victorleblais** [comment]
Just as a note: You can now rename cities for a fee (quite expensive) in the experimental branch of the game.

**Reply by JezzyBelle** [reply]
I noticed.  Perfect price too in my opinion.

**Reply by JezzyBelle** [reply]
Oh Wait a minute there. LOL. When I started this map I think it was going to let me change the name of my starter city for 500,000. I may be wrong on that.. But, even if it was 5 million... As a starter city that felt like a large price to pay while I had better things to spend my money on. Now, I had a little cash and figured, eh, 500k is a lot of cash but hey, nice to be able to finally afford it. Price to change the name of my city is now 294 million, nearly 1/3 of a billion dollars?  Yeah... Can we adjust that rate a bit and maybe cap it at say 5 million? I mean, this town only has a population of 6714.  

In theory, it would be far easier and cheaper for me to find some small town in another region for 12 million(region), pay 1.2 million to rename the town, then, rebuild everything from scratch and remove my presence in that other city entirely. I could even shut down entirely and sustain huge losses and still come out of it cheaper.  Crazy.. lol

**Reply by JezzyBelle** [reply]
Thought about it some more and am still shocked at the current price to rename my home city. Looked at my net worth. They want more than my company is worth by 100 million. I would suggest a formula based upon population but maybe capped at 1 or 2 million per 1,000 pop. Or something more attainable.. Right now, in my home city, it looks like they want to charge me 50,000 per person in the city.. Almost every other small city seems to be close to 6,000 per person.
At least in the case of my home city, you would think they would love me for having supplied them and made them my hub.. supplying jobs and all.  IN a home city,. they may even want to celebrate our mutual successes and offer to rename it in hy honor even.   LOL. Anyway. it would be an accomplishment for sure.... but I see an accomplishment that is attainable and or, seemingly worth it.  Anyway.. thanks for looking at this!

**Reply by victorleblais** [reply]
I totally agree that the price seems way too high and should be capped to maybe something like 5 millions or maybe a percentage of your company's worth.

The feature is still part of the experimental branch so it will probably be adjusted. Thanks a lot for your valuable and detailed feedback by the way !
